pkg/util/kernel/find_headers.go

```diff
@@ -461,7 +460,7 @@ func submitTelemetry(result headerFetchResult, client statsd.ClientInterface) {
 
 	khdTags := append(tags,
 		fmt.Sprintf("result:%s", resultTag),
-		fmt.Sprintf("reason:%s", model.KernelHeaderFetchResult(result).String()),
+		fmt.Sprintf("reason_code:%d", result),
```

Comment:
Can we copy over the int->string mapping from `agent-payload` instead? The numbers are going to be unusable in our metrics.

Reply:
Sure, done in [copy kernelHeaderFetchResult_name](https://github.com/DataDog/datadog-agent/pull/29478/commits/5d9dca144e81514169dd1dc310130806d9329b8f)